Sources/Fuzzilli/Evaluation/ProgramCoverageEvaluator.swift

```diff
@@ -121,7 +187,49 @@ public class ProgramCoverageEvaluator: ComponentBase, ProgramEvaluator {
             return true
         }
     }
-    
+
+    func resetEdge(_ edge: UInt32) {
+        resetCounts[edge] = (resetCounts[edge] ?? 0) + 1
```

Comment:
Can you add a comment here `// TODO see if we want to count the number of non-deterministic edges and expose them through the fuzzer statistics (if deterministic mode is enabled)`. No need to implement that, but I think it could be interesting to measure that